gitlab/golang/stages/30-tests/go.yaml

```diff
@@ -1,18 +1,28 @@
 include:
-  - remote: 'https://raw.githubusercontent.com/rios0rios0/pipelines/main/gitlab/golang/abstracts/go.yaml'
+- remote: 'https://raw.githubusercontent.com/rios0rios0/pipelines/main/gitlab/golang/abstracts/go.yaml'
 
-test:all:
+.test:all:
   extends: '.go'
   stage: 'tests'
   script:
-    - !reference [ .scripts-repo, before_script ]
-    - $SCRIPTS_DIR/global/scripts/golang/test/run.sh
+  - !reference [.scripts-repo, before_script]
+  - $SCRIPTS_DIR/global/scripts/golang/test/run.sh
   coverage: '/total.*?([0-9]{1,3}[.][0-9]{0,2})%/'
   artifacts:
     when: 'always'
     paths:
-      - 'coverage.*'
+    - 'coverage.*'
     reports:
       coverage_report:
         coverage_format: 'cobertura'
         path: 'coverage.xml'
+
+.test:build:
+  extends: '.go'
+  stage: 'tests'
+  script:
+  - mkdir ./bin
+  - go build -o ./bin ./cmd/...
```

Comment:
I think it's better to create a couple of lines before this to detect what is the folder with go files. It could be internal as well. @k4yt3x what do you think about that? Give me your thoughts .

Reply:
Or you could `go build -o ./bin ./...`?